dataland-frontend/tests/component/components/resources/dataTable/MultiLayerDataTableWithEsgQuestionnaire.cy.ts

```diff
@@ -34,8 +34,8 @@ describe("Component Test for the ESG Questionnaire view Page with its componenet
       data: preparedFixtureForTest.t,
     } as CompanyAssociatedDataEsgQuestionnaireData);
     mountEsgQuestionnaireFrameworkFromFakeFixture([preparedFixtureForTest]);
-    getSectionHead("Umwelt").click();
-    getSectionHead("Treibhausgasemissionen").click();
+    getSectionHead("Umwelt");
+    getSectionHead("Treibhausgasemissionen");
```

Comment:
both get statements could just be deleted as they serve no purpose

Reply:
.should("exist") added